src/java/org/apache/cassandra/dht/BootStrapper.java

```diff
@@ -154,9 +154,8 @@ public void onFailure(Throwable throwable)
      * otherwise, if num_tokens == 1, pick a token to assume half the load of the most-loaded node.
      * else choose num_tokens tokens at random
      */
-    public static Collection<Token> getBootstrapTokens(final TokenMetadata metadata) throws ConfigurationException
+    public static Collection<Token> getBootstrapTokens(final TokenMetadata metadata, Collection<String> initialTokens) throws ConfigurationException
```

Comment:
I think we should still keep one function with the signature `public static Collection<Token> getBootstrapTokens(final TokenMetadata metadata) throws ConfigurationException` that calls `getBootstrapTokens(metadata, DatabaseDescriptor.getInitialTokens())` so that we can still use config if needed.

Reply:
I _think_ we're ok here, as this method is only used in `joinTokenRing`, and we pass in the config values for initial tokens: https://github.com/palantir/cassandra/pull/262/files#diff-9bf2c26bc294ef9085e16bf287490223665eaa2eb8ec24bcf5bd8653c713644bR1039